inc/thorvg.h

```diff
@@ -958,7 +958,7 @@ class TVG_API Shape final : public Paint
      *
      * @return Result::Success when succeed, Result::FailedAllocation otherwise.
      */
-    Result stroke(uint8_t r, uint8_t g, uint8_t b, uint8_t a = 255) noexcept;
+    Result strokeFill(uint8_t r, uint8_t g, uint8_t b, uint8_t a = 255) noexcept;
```

Comment:
I think it is good if the parameters and names are intuitive. What about `strokeColor?'

Reply:
I intentionally aligned with Shape::fill(uint8_t r, uint8_t g, uint8_t b, uint8_t a = 255);

Reply:
fill is also acceptable to me, Otherwise, we could change `Shape::fill()` to `Shape::color()` as well. It would result in significant changes, and I would prefer to minimize these changes.